CLAUDE.md

```
@@ -42,18 +42,126 @@ pnpm format:check           # Prettier check
+
+| Layer      | File                               | Class                            |
+| ---------- | ---------------------------------- | -------------------------------- |
+| Controller | `admin-registration.controller.ts` | `AdminRegistrationController`    |
+| Service    | `admin-registration.service.ts`    | `AdminRegistrationService`       |
+| Repository | `registration.repository.ts`       | `RegistrationRepository`         |
+| Error      | `registration.errors.ts`           | `SlotConflictError`              |
+| Mapper     | `mappers.ts`                       | `toPlayer()`, `toRegistration()` |
+
+## Data Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)